### PR TITLE
updates readme with project/tooling section guidelines

### PR DESCRIPTION
**Pull request by nellshamrell**
As outlined in #3639 

Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,10 +59,26 @@ What we are generally NOT looking for includes:
...
 * Anything that requires information to be shared/captured (like an email address) in order to access
 
+**Projects/Tooling Updates**
+
+There are further guidelines for the Projects/Tooling Updates Section
+
+We include:
+
+* Updates on tooling when in long form or framed as a tutorial (this can be through a blog, through GitHub, through a newsletter, and more) - it must have a high amount of Rust specific info (examples in Rust, notes on things learned about Rust in the process of creating/updating the project, etc.)
+* Updates on tooling that call out specific contributors - it is wonderful to highlight all the great people contributing to Rust OSS projects
```
Comment (U007D) [inline README.md:72]: > Updates on tooling that call out specific contributors - it is wonderful to highlight all the great people contributing to Rust OSS projects

Can someone use attribution as a "pass" for not including Rust-specific info?

File: README.md [hunk 2]
```diff
@@ -59,10 +59,26 @@ What we are generally NOT looking for includes:
...
+
+We do not include:
+* Updates that are solely a change log without details on the changes, guides to using the changes, etc.
+* Links that are solely to a GitHub repo or crate on crates.io. While we would love to include these, there are too many being created/updated every week for us to include them all. We encourage you to write up an introduction to your project with examples, information you learned through creating the project, etc.
+
+Notes:
+* A small description of the project or the update in your link is fine (for example: FooBar 1.0: adding support for Baz)
```
Comment (U007D) [inline README.md:79]: > in your link is fine

is encouraged? (helps more people understand and/or remember what the crate does)

Comment (nellshamrell) [reply README.md:79]: It makes sense to change that to "is encouraged?" - changing that

File: README.md (hunk 1 above)
Comment (nellshamrell) [reply README.md:72]: Er, no. I will clarify.

**Comment by nellshamrell** [comment]
Messed up the git history on this one, closing and re-opening a new PR.

**Pull request closed without merging**